test/modellib/ConvModel.cpp

```diff
@@ -119,6 +119,19 @@ bool Conv2DLibBuilder::build() {
   return true;
 }
 
+bool Conv2DLibBuilder::prepareInputs(float dataRange) {
+  constexpr int num = 2;
+  OMTensor **list = (OMTensor **)malloc(num * sizeof(OMTensor *));
+  if (!list)
+    return false;
+  list[0] =
+      omTensorCreateWithRandomData<float>({N, C, H, W}, -dataRange, dataRange);
+  list[1] = omTensorCreateWithRandomData<float>(
+      {C, C, kH, kW}, -dataRange, dataRange);
+  inputs = omTensorListCreateWithOwnership(list, num, true);
+  return inputs && list[0] && list[1];
+}
+
 bool Conv2DLibBuilder::prepareInputs() {
```

Comment:
can you redefine this one as calling the new prepareInputs with `dataRange = 1.0` please?

Reply:
Done.

Reply:
great, less redundant code